apps/events-helsinki/src/hooks/useEventsRHHCConfig.tsx

```diff
@@ -75,6 +79,8 @@ export default function useEventsRHHCConfig(args: {
         VenueCardContent: (props: any) => (
           <VenueDetails {...(props as VenueDetailsProps)} />
         ),
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        CheckMarkIcon: (props: any) => <HelsinkiCityOwnedIcon {...props} />,
```

Comment:
Should this be something like "HelsinkiCityOwnedIcon" instead of "CheckMarkIcon"?